app/Helpers/Dusk.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Illuminate\Support\HtmlString;
+
+if (! function_exists('dusk')) {
+    function dusk(string $selector): ?HtmlString
```

Comment:
@brendt I tried with this helper function, it works on html elements but not on blade components :(. 
This is the error I get when I attempt to use this helper on a blade component. 
```balde
        <x-home.title {{dusk('test')}}>
            {{ __('Open RFCs') }}
        </x-home.title>
```
![image](https://github.com/brendt/rfc-vote/assets/5923843/3d1ddb8e-5252-460e-ac92-55a6dbf64f71)

Not really sure how else to approach this at this point :), I'm willing to work more on this but I would need some ideas/suggestions.

Reply:
Well I think the dusk function should be move to the component itself, no? Otherwise you'd have to repeat it every time you add the component.

Reply:
Yep, but we need to keep this limitation in mind, if that's ok, I can try to use it and see how it goes.

Reply:
I think that's ok :)